src/column.rs

```diff
@@ -15,6 +16,7 @@ pub enum FunctionExpression {
     Max(FunctionArguments),
     Min(FunctionArguments),
     GroupConcat(FunctionArguments, String),
+    Concat(Vec<FieldDefinitionExpression>),
```

Comment:
The use of `FieldDefinitionExpression` here seems potentially troublesome. `FieldDefinitionExpression` encompasses variants `All` (which is equivalent to `*` in a select query) and `AllInTable` (which is `table.*`), but neither of these work with MySQL's `concat()` function:
```
mysql> select concat (email, roles) from ContactInfo;
+-----------------------+
| concat (email, roles) |
+-----------------------+
| malte@cs.brown.edu5   |
+-----------------------+
1 row in set (0.01 sec)

mysql> select concat (ContactInfo) from ContactInfo;
ERROR 1054 (42S22): Unknown column 'ContactInfo' in 'field list'
mysql> select concat (ContactInfo.*) from ContactInfo;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '*) from ContactInfo' at line 1
mysql> select concat (*) from ContactInfo;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '*) from ContactInfo' at line 1
```

In other words, we should only allow columns and literals here. Perhaps we need a variant of `FieldDefinitionExpression` that only supports column names and literals.